.github/workflows/build_and_test.yaml

```diff
@@ -47,6 +47,10 @@ jobs:
     # build
     - run: make build-multiarch
 
+    # conformance
+    - name: Run Conformance Tests
+      run: make conformance
```

Comment:
since no command above is pushing this image to the hub, will this command auto load the built image (default IMAGE=envoyproxy/gateway-dev TAG=commit-short-sha) into the cluster ?

Reply:
The certgen job was not creating the secret that EG requires at startup. 741b1efd31e3578e1168094a4be05ecff32cb3fb updates the `kube-install-image` target so the certgen job pulls the built EG image from kind nodes instead of the EG docker repo.